Formula/rex.rb

```diff
@@ -206,9 +214,16 @@ class Rex < Formula
   end
 
   def install
+    # First prevent trying to sniff around elsewhere for required modules.
+    ENV.delete("PERL5LIB")
+
     ENV.prepend_create_path "PERL5LIB", libexec/"lib/perl5"
     ENV.prepend_path "PERL5LIB", libexec/"lib"
 
+    # If you're using a new enough perl it'll blow up & require your
+    # confirmation that you know what you're doing. Skip that.
+    ENV["PERL_CANARY_STABILITY_NOPROMPT"] = "1"
```

Comment:
Okay, confirmed: the guy behind that warning is being evangelical. Have a read of the [first few entries in his FAQ](http://blog.schmorp.de/2015-06-06-stableperl-faq.html), and you'll quickly get the gist.

Also:
> _"Simply because the new set of perl 5 porters (the people maintaining the standard version of perl 5) have an abysmal track record of providing stable releases. … I plan to write something in more detail about this, using references, but at the moment, I am too depressed by this and in any case, **I have to save perl**, so stay tuned for a later article on this subject."_

lolwat.